### PR TITLE
Documentation - Manual: portuguese added

### PR DESCRIPTION
**Pull request by geankaminski**
Related issue: #24375

Portuguese translation of the manual section

### CODE REVIEW

File: test/package-lock.json [hunk 1]
```diff
@@ -25,20 +25,20 @@
       "version": "0.142.0",
       "license": "MIT",
       "devDependencies": {
-        "@babel/core": "^7.18.2",
```
Comment (Mugen87) [inline test/package-lock.json:28]: This file should also be removed from the PR.

File: build/three.cjs [hunk 2]
```diff
@@ -28495,7 +28495,7 @@ class LoadingManager {
 
 }
 
-const DefaultLoadingManager = new LoadingManager();
```
[the remark this answers is not in the document]
Comment (geankaminski) [reply build/three.cjs]: ops, sorry. files removed!

[the remark this answers is not in the document]
Comment (geankaminski) [reply build/three.cjs]: ops sorry. build files and package-lock.json from /test removed from PR

**Comment by sunag** [comment]
The translation looks good to me!

File: docs/list.json [hunk 3]
```diff
@@ -1,9 +1,7 @@
 {
-
+
```
Comment (Mugen87) [inline docs/list.json:2]: It seems this file was unnecessarily formatted. Can you please revert these changes?

File: docs/index.html [hunk 4]
```diff
@@ -36,6 +36,7 @@ <h1><a href="https://threejs.org">three.js</a></h1>
 						<option value="ko">한국어</option>
 						<option value="zh">中文</option>
 						<option value="ja">日本語</option>
+						<option value="pt-br">pt-br</option>
```
Comment (LeviPesin) [inline docs/index.html:39]: Shouldn't the text be the translation of "Brazilian Portugues" to Portuguese rather than just `pt-bt` string?

**Comment by LeviPesin** [comment]
https://github.com/geankaminski/three.js/blob/74f93aa9f36b8c5f7ad05fec8a77a32f5444cad0/docs/index.html#L90

Shouldn't `pt-br` also go here?

File: docs/list.json (hunk 3 above)
Comment (geankaminski) [reply docs/list.json]: I think this is resolved!

File: docs/index.html (hunk 4 above)
Comment (geankaminski) [reply docs/index.html:39]: I think this is the pattern, like the other languages in the file

**Comment by geankaminski** [comment]
> https://github.com/geankaminski/three.js/blob/74f93aa9f36b8c5f7ad05fec8a77a32f5444cad0/docs/index.html#L90
> 
> Shouldn't `pt-br` also go here?

Yes! I will fix this

File: docs/index.html (hunk 4 above)
Comment (LeviPesin) [reply docs/index.html:39]: I think the pattern is using the language name translated to that language.

Comment (geankaminski) [reply docs/index.html:39]: Not necessarily, we have "en" and "ar" in the first options. Also, i think "português brasileiro" would be too long compared to the others options

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!
